Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml

```
@@ -66,7 +66,7 @@
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltSyndicate
   name: syndicate overear-headset
-  description: A syndicate headset that can be used to hear all radio frequencies. Protects ears from flashbangs.
+  description: A Syndicate overear headset, designed for work in special operations, has access to the Syndicate frequencies, sadly, not Nanotrasen ones.
```

That's, a lot, of commas.

Think it's better just adjusting the original.

```suggestion
  description: An updated, modular syndicate intercom that fits over the head and takes encryption keys. Protects ears from flashbangs.
```

Frequencies can change and are available via detailed examine anyway.